jkube-kit/generator/webapp/src/test/java/org/eclipse/jkube/generator/webapp/AppServerAutoDetectionTest.java

```diff
@@ -121,18 +121,18 @@ public void testDefaultServer() throws IOException {
             .build()).build();
 
         AppServerHandler appServerHandler = new AppServerDetector(generatorContext).detect(null);
-        assertEquals("tomcat", appServerHandler.getName());
+        assertThat(appServerHandler.getName()).isEqualTo("tomcat");
     }
 
     private void assertAppServerDescriptorApplicability(Object[] descriptors) throws IOException {
         for (int i = 0; i < descriptors.length; i += 2) {
             String descriptor = (String) descriptors[i];
             boolean expected = (boolean) descriptors[i + 1];
 
-            File appDir = folder.newFolder("webapp" + i);
-            assertTrue(new File(appDir, "META-INF/").mkdirs());
-            assertTrue(new File(appDir, "WEB-INF/").mkdirs());
-            assertTrue(new File(appDir, descriptor).createNewFile());
+            File appDir = Files.createDirectory(folder.resolve("webapp" + i)).toFile();
+            assertThat(new File(appDir, "META-INF/").mkdirs()).isTrue();
+            assertThat(new File(appDir, "WEB-INF/").mkdirs()).isTrue();
+            assertThat(new File(appDir, descriptor).createNewFile()).isTrue();
```

Comment:
As commented [here](https://github.com/eclipse/jkube/issues/1563#issuecomment-1203719846), these assertions are not necessary if you use the nio Files utility.
Also, consider removing the slash from the directory paths.